### ISSUE TITLE
Not detecting VALORANT process

### ISSUE THREAD

**Issue by didotb** [colinhartigan/valorant-rpc#16]
Description:
After opening valorant-rpc version 2.1, it tries to open valorant game which it does.
I login to riot, then the game launches.

Expected Result:
valorant-rpc should detect that the game has launched or was already launched, and the RPC will start.

Actual Result:
valorant-rpc keeps checking if the game has launched till the 120 second mark (or whatever is set for the launch-timeout in config.json file), then closes for it doesn't detect the game process

P.S. I don't know where the debug log file is if any. I changed "debug" to "true" in the config.json file, but I don't know where the file is located if there is one.

**Comment by colinhartigan** [comment]
check task manager for RiotClientServices.exe and VALORANT-Win64-Shipping.exe when running the game?

**Comment by didotb** [comment]
yep it was.
I also tried different varieties of steps on opening the game / rpc app to see if there's a certain way that the app could detect the game, but no dice.

This was only a problem with the version 2.1 using the installer. I uninstalled that, and tried cloning the main branch instead and it works.

Since then I've been cloning the main branch instead, and it works.
I'll try the latest update whenever I finish my work sometime this week, so I can verify if it's still a thing.

EDIT:
I cloned the main branch since I tried reviewing the code if there's somehow a weird problem with the files within the installer versus the main branch, and I found nothing.
It was honestly weird that happened and I was surprised there were no issues about it already.

P.S. I tried it with 3 other people and only I got the issue. So I tried restarting twice but no dice
